### PR TITLE
Support to get CMYK color in Image#pixel_color

### PR DESCRIPTION
**Pull request by Watson1978**
This patch will support to get CMYK color in Image#pixel_color.

Related to #871

### CODE REVIEW

File: ext/RMagick/rmpixel.c [hunk 1]
```diff
@@ -659,15 +659,15 @@ Pixel_from_MagickPixel(const MagickPixel *pp)
     Pixel *pixel;
 
     pixel          = ALLOC(Pixel);
-    pixel->red     = ROUND_TO_QUANTUM(pp->red);
-    pixel->green   = ROUND_TO_QUANTUM(pp->green);
-    pixel->blue    = ROUND_TO_QUANTUM(pp->blue);
+    pixel->red     = pp->red;
```
Comment (Watson1978) [inline ext/RMagick/rmpixel.c]: `Quantum` type is alias of `MagickFloatType`.
`MagickRealType`  type is alias of `MagickDoubleType`.

So. there is no needed to round the value.

Comment (dlemstra) [reply ext/RMagick/rmpixel.c]: `ROUND_TO_QUANTUM` make sure that the value is within the quantum rang. Are you really sure that we don't need it?

Comment (Watson1978) [reply ext/RMagick/rmpixel.c]: If I want 0.0 value, however, `ROUND_TO_QUANTUM()` adds 0.5 value...

https://github.com/rmagick/rmagick/blob/79f505762bdc47a3426246e5379ad5753deb2099/ext/RMagick/rmagick.h#L102

Looks for me that `ROUND_TO_QUANTUM()` is unnecessary in here.

Comment (Watson1978) [reply ext/RMagick/rmpixel.c]: Or, we should fix `ROUND_TO_QUANTUM` like

```
 #define ROUND_TO_QUANTUM(value) ((Quantum) ((value) > (Quantum)QuantumRange ? QuantumRange : value)) 
```

**Pull request merged**